products/wordpress.md

```diff
@@ -33,196 +39,196 @@ identifiers:
 # Support PHP version can be found on https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/.
 releases:
 -   releaseCycle: "6.4"
-    supportedPHPVersions: "7.0, 7.1, 7.2, 7.3, 7.4, 8.0, 8.1, 8.2, 8.3"
+    supportedPHPVersions: "7.0 - 7.4, 8.0 - 8.3"
```

Comment:
```suggestion
    supportedPHPVersions: "7.0 - 8.3"
```

Just a suggestion, but looking at https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/ `<smallest version> - <biggest version>` always form a single range. This way the column would take less space.

Reply:
Good suggestion, I'll update the other entries accordingly.